### ISSUE TITLE
Autodetect uploaded document type

### ISSUE THREAD

**Issue by tpluscode** [inception-project/inception#4666]
**Is your feature request related to a problem? Please describe.**

When uploading PDF documents one-by-one, it is quite annoying that the type needs to be switched to PDF every time. While some other formats all share the XML or JSON general type, PDFs can hardly be confused with anything else. It would be nice it weren't necessary to select the format every time.

Currently, keeping the default selection "Plain text" usually returns an error about "too many sentences" or something like that

**Describe the solution you'd like**

Having selected a PDF document to import, or multiple documents all of which are PDFs, the format would automatically change to match the type

**Describe alternatives you've considered**

An "autodetect" option, selected as default "format" would also work. That could potentially be more useful, being able to distinguish the other formats which are all JSON, etc, albeit requiring more work

**Comment by reckart** [comment]
You can select multiple files in the upload dialog. Typically, you can upload up to 999 files or whatever limit your browser imposes.

**Comment by tpluscode** [comment]
Yes, I do mention that too. I only meant that I usually have only one document to upload at a time
